tiered-cart-promotions/README.md

```diff
@@ -24,6 +24,228 @@ The demo is running with a [Sandbox project](https://docs.voucherify.io/docs/tes
 
 <img width="1189" alt="Screenshot 2022-08-03 at 14 52 39" src="https://user-images.githubusercontent.com/77458595/182775604-db10d656-f9f8-410f-ad74-6aa746ba4727.png">
 
+## Creating a "Reward Promotion" campaign<a id="creating-campaign"></a>
+
+### First of all, it is required to add missing ```validation_rules```:
+- Change X-App-Id to your Application ID.
+- Change X-App-Token to your Secret Key.
+- Keep the returned validation_rule id after executing each request. It will be required for the main request.
+
+```
+curl --location --request POST 'https://api.voucherify.io/v1/validation-rules' \
```

Comment:
`missing-campaign.js` instead of curls?

Reply:
up

Reply:
We do not need to describe curl details if we have a JS script. Just leave a information that you need to run a script if you have missing campaigns